tests/lib/rules/indent.test.js

```diff
@@ -0,0 +1,180 @@
+/**
+* @fileoverview Check indentation at the beginning and end of a function call
+*/
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require('../../../lib/rules/indent'),
+    formatCode = require('../../code-helper'),
+    RuleTester = require('eslint').RuleTester,
+    ruleTester = new RuleTester(),
+    validExample, switchTest, constExample,
+    invalidExample, invalidExample2, invalidExample3, constInvalid;
+
+validExample = formatCode(
+   'var a = 1,',
+   '    b = 2,',
+   '    c = 3;',
+   '',
+   'function test() {',
+   '   a()',
+   '      .b(function() {',
+   '         var a = 1,',
+   '             b = 2;',
+   '',
+   '         return a()',
+   '            .b(function() {});',
+   '      });',
+   '}',
+   '',
+   'function test2() {',
+   '   var arr = [ 1, 2, 3 ],',
+   '       noInit, andAgain;',
+   '',
+   '   if (noInit) {',
+   '      var a = 1,',
+   '          b = 2;',
+   '   }',
+   '}'
+);
+
+invalidExample = formatCode(
+   'function test() {',
+   '   var a = 1,',
+   '      b = 2;',
+   '',
+   '   return a + b;',
+   '}'
+);
+
+invalidExample2 = formatCode(
+   'var a = 1,',
+   '    b = 2;',
+   '',
+   'function badIndent() {',
+   '  a = b;',
+   '}'
+);
+
+invalidExample3 = formatCode(
+   'function test() {',
+   'notindented();',
+   '}'
+);
+
+switchTest = formatCode(
+   'switch (a) {',
+   '   case 1:',
+   '      break;',
+   '   default:',
+   '      c = a + b',
+   '}'
+);
+
+constExample = formatCode(
+   'const A = 1,',
+   '      B = 2;'
+);
+
+constInvalid = formatCode(
+   'const A = 1,',
+   '   B = 2;',
+   '',
+   'const C = 3,',
+   '    D = 4;'
+)
```

Comment:
Missing a semi-colon here. Please run `grunt standards` before issuing a PR to ensure that the repo meets its own standards.